packages/directlinespeech/src/createAdapters.js

```diff
@@ -155,6 +166,31 @@ export default async function create({
     }, TOKEN_RENEWAL_INTERVAL);
   }
 
+  // Renew token per interval.
+  if (enableInternalHttpSupport) {
+    const interval = setInterval(async () => {
+      // #2660 If the connector has been disposed, we should stop renewing the token.
+
+      // TODO: We should use a public implementation if Speech SDK has one related to "privIsDisposed".
+      if (dialogServiceConnector.privIsDisposed) {
+        clearInterval(interval);
```

Comment:
It will still continue to refresh the token even the connector is disposed.

Did you actually try out this code path?

Please make sure there is a test case targeting this code path.

Reply:
this is based off the renewal logic for Direct Line Speech token right above this. Why is this not a concern for DirectLine Speech but a concern for DirectLine?

Can you point me to some tests for the renewal token logic? i'll create them accordingly for DirectLine as well.

Reply:
@compulim ping...